mods/cnc/maps/gdi03/rules.yaml

```diff
@@ -90,7 +90,9 @@ SAM:
 	Building:
 	Power:
 		Amount: -10
-	-Capturable:
+	-ExternalCapturable:
+        -CaptureCompleteTime: 5
```

Comment:
Does this break it? Because those are tabs. Notepad++ apparently decided to screw everything up for... no reason. It looked fine and didn't show any red spaces before its PR.